bucket/vieb.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "4.1.0",
+    "description": "Vieb is the Vim Inspired Electron Browser",
+    "homepage": "https://vieb.dev/",
+    "license": "GPL-3.0-or-later",
+    "hash": "1c958e4cba47248213f4b9ede909ea6f57cf24967e953fceb489b5342b1f6f88",
```

Comment:
Hash is checked after the URL is downloaded. Does not make sense to have the hash above URL

Reply:
Fixed